sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -71,18 +73,26 @@ def _load_model(
   try:
     logging.info(
         "Loading state_dict_path %s onto a %s device", state_dict_path, device)
-    state_dict = torch.load(file, map_location=device)
+    if not use_torch_script_format:
```

Comment:
>> My inclination after thinking more is that we should not do this since (a) it gives the users less safety from accidental errors, and (b) it is potentially constraining if we accept loading new model types that don't use state_dict_path or model_class in the future.

Is this for the comment to make `model_class` as None and inferring from it? or is it regarding adding the parameter `use_torch_script_format`?

Reply:
This comment is a no-op. I was considering the idea of not having a `use_torch_script_format` parameter and just inferring whether we should do that based on if `model_class` was set. My second comment here was saying that I don't actually think its a good idea.

Mostly just wanted to call it out since it was a sticking point on my first review pass.